.github/workflows/connect-release-init.yml

```diff
@@ -56,11 +51,11 @@ jobs:
         with:
           node-version-file: ".nvmrc"
 
-      - name: Run @trezor/connect release trezor.io
```

Comment:
this job should be created ideally after the PR created by the 1st one was merged. @vdovhanych if you have any idea how to achieve it, we could do it now. If not, we can live with it as it is for some more time I guess.